evap/evaluation/tests/test_views.py

```diff
@@ -246,3 +248,44 @@ def test_notebook(self):
 
         user.refresh_from_db()
         self.assertEqual(user.notes, self.note)
+
+
+class TestResetEvaluation(WebTestStaffMode):
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.manager = make_manager()
+        cls.semester = baker.make(Semester, results_are_archived=True)
+
+    def reset_from_x_to_new(self, x, success_expected: bool) -> None:
+        evaluation = baker.make(Evaluation, state=x, course__semester=self.semester)
+
+        semester_overview_page = self.app.get(
+            reverse("staff:semester_view", args=[self.semester.pk]), user=self.manager, status=200
+        )
+
+        form = semester_overview_page.forms["evaluation_operation_form"]
+
+        form["evaluation"] = [evaluation.pk]
+
+        confirmation_page = form.submit(name="target_state", value=str(Evaluation.State.NEW))
```

Comment:
(opinion): If you add `cls.url = reverse("staff:semester_view", args=[self.semester.pk])` in `setUpTestData`, this can be compacted to:

```suggestion
    def reset_from_x_to_new(self, x, success_expected: bool) -> None:
        evaluation = baker.make(Evaluation, state=x, course__semester=self.semester)

        semester_overview_page = self.app.get(self.url, user=self.manager, status=200 )
        form = semester_overview_page.forms["evaluation_operation_form"]
        form["evaluation"] = [evaluation.pk]
        confirmation_page = form.submit(name="target_state", value=str(Evaluation.State.NEW))
```

which I like more -- imo multiple single-statement "blocks" don't really help with readability as they don't provide semantical grouping.